h/models/__init__.py

```diff
@@ -39,6 +40,7 @@
     'Annotation',
     'AnnotationModeration',
     'AuthClient',
+    'AuthzCode',
```

Comment:
Oh no! I missed that one. I specifically went over every changed file and ran `isort`, but I didn't look at this tuple!